### ISSUE TITLE
Action fails after upgrading flyctl version to 0.2.73

### ISSUE THREAD

**Issue by sheerlox** [superfly/flyctl-actions#61]
The action fails without any error log (even in debug mode) since upgrading to [flyctl `v0.2.73`](https://github.com/superfly/flyctl/releases/tag/v0.2.73):

![image](https://github.com/superfly/flyctl-actions/assets/11234273/a2bba913-2af1-4e13-91dd-15a3fbcc1aab)

Workflow file:
```yml
name: Deploy Staging

on:
  workflow_dispatch:
  workflow_run:
    workflows: ["Release"]
    types:
      - completed

jobs:
  deploy-staging:
    name: Deploy app
    runs-on: ubuntu-latest
    if: ${{ github.event.workflow_run.conclusion == 'success' || github.event_name == 'workflow_dispatch' }}
    concurrency: deploy-staging-group
    steps:
      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
        with:
          fetch-depth: 0
          fetch-tags: true

      - name: Create .env file
        run: cp .env.example .env

      - name: Output app revision
        run: make get-revision

      - uses: superfly/flyctl-actions/setup-flyctl@master
        with:
          version: 0.2.73 # renovate: datasource=github-releases depName=flyctl packageName=superfly/flyctl versioning=semver

      - run: flyctl deploy --remote-only --config fly.staging.toml -e APP_REVISION="$(make get-revision)"
        env:
          FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN_STAGING }}
```

Downgrading to `v0.72.2` solves the issue:
![image](https://github.com/superfly/flyctl-actions/assets/11234273/19be4e3f-f850-4974-ab74-1600e929976a)

**Comment by sheerlox** [comment]
@rubys looks like you forgot to add the binaries to the release :smile:

**Comment by rszalski** [comment]
Ran into the same issue with `v0.2.80`. It doesn't seem like this has been resolved. Not sure what is the issue but I just downgraded to `v0.2.72` as a workaround.

**Comment by sheerlox** [comment]
@rszalski that's weird, because the binaries are present on the `v0.2.80` release as mentioned in the main issue (https://github.com/superfly/flyctl/issues/3681#issuecomment-2206545883).

**Comment by sheerlox** [comment]
Can confirm there's still an issue.

![image](https://github.com/superfly/flyctl-actions/assets/11234273/4344f7d0-6ab0-48b7-9e48-7ef50242b0dc)

**Comment by TheRealFlyingCoder** [comment]
Confirming this issue is still present on the newest patches up to 0.2.84.

**Comment by clouvet** [comment]
This should be fixed now. We had an issue with the package naming during automatic release through GitHub Actions that we resolved: https://github.com/superfly/flyctl/pull/3721. 

@sheerlox and @TheRealFlyingCoder, if you can confirm that the problem is solved for 0.2.85 and up that'd be super helpful.

**Comment by sheerlox** [comment]
Problem solved, thanks!

![image](https://github.com/superfly/flyctl-actions/assets/11234273/294b114b-0252-4bf6-b645-af19d34d78d4)
